### PR TITLE
object_torch2, object_blkobj, and object_geldb

### PR DESCRIPTION
**Pull request by petrie911**
Trying my hand at this object decomp thing. Also cleaned up a few things with GeldB and got rid of its relocs, which hadn't happened when it was matched.

### CODE REVIEW

**Comment by fig02** [comment]
neat stuff
a few things:
- pr changes to zap, dontinclude them here
- you dont need a gitkeep file, there just needs to be one in the objects folder which is already there
- leading zeros on the segment number apparently cause issues in some cases according to zel, so just put 6
- you can remove the commented lines in the spec
- can you add empty newlines at the end of the XMLs
- the suffixes we are going with so far are:
`Anim`, `Skel`, `DL`, `Col`. and they need to be at the end of the name.

**Comment by fig02** [comment]
some of my comments mentioned above still need to be addressed

**Comment by petrie911** [comment]
Should be good now.

**Review by fig02 — APPROVED** [review]

File: assets/xml/objects/object_geldb.xml [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+        <Animation Name="gGerudoRedSpinAttackAnim" Offset="0x0F5C"/>
+        <Animation Name="gGerudoRedDamageAnim" Offset="0x2280"/>
+        <Animation Name="gGerudoRedDefeatAnim" Offset="0x1E10"/>
+        <Animation Name="gGerudoRedBlockAnim" Offset="0x1578"/>
+        <Animation Name="gGerudoRedStandAnim" Offset="0xB07C"/>
+
+        <!-- Red-clothed Gerudo eye textures -->
+        <Texture Name="gGerudoRedEyeOpenTex" Format="ci8" Width="32" Height="32" Offset="0x5FE8"/>
+        <Texture Name="gGerudoRedEyeHalfTex" Format="ci8" Width="32" Height="32" Offset="0x65A8"/>
+        <Texture Name="gGerudoRedEyeShutTex" Format="ci8" Width="32" Height="32" Offset="0x6D28"/>
```
[the remark this answers is not in the document]
Comment (petrie911) [reply assets/xml/objects/object_geldb.xml:22]: I'm not sure what you mean by this.

[the remark this answers is not in the document]
Comment (Roman971) [reply assets/xml/objects/object_geldb.xml:22]: The png file generated by extracting the texture needs a name, which we specify with the `OutName` attribute.
For example: `<Texture Name="gGerudoRedEyeOpenTex" OutName="gerudo_red_eye_open" ... />`

**Review by fig02 — APPROVED** [review]

**Review by Roman971 — APPROVED** [review]

**Pull request merged**